databuilder/tables/beta/tpp.py

```diff
@@ -314,6 +314,84 @@ class isaric_raw(EventFrame):
         description="Onset date of first/earliest symptom.",
     )
 
+    # Clinical characteristics
+    chrincard = Series(
+        str,
+        description="Chronic cardiac disease, including congenital heart disease (not hypertension).",
+    )
```

Comment:
These need a categorical constraint on them so they get typed correctly as categorical variables. Something like:
```suggestion
    chrincard = Series(
        str,
        description="Chronic cardiac disease, including congenital heart disease (not hypertension).",
        constraints=[
            Constraint.Categorical(["Yes", "No", "Unknown"])
        ],
    )
```